mitodl/mitxonline
# Addded Discount models

#### Pre-Flight checklist


#### What are the relevant tickets?
Fixes #309 

#### What's this PR do?
* Adds in models for Discounts, User Discounts, and Redemptions
* Adds bindings to dj-admin so those can be viewed/managed through that

#### How should this be manually tested?
Run migrations, then test via django admin

- This doesn't use reversion but that might not be a bad idea; should be pretty easy to add in.
